spec/features/visitor_sees_all_the_user_spec.rb

```diff
@@ -34,7 +34,11 @@
   end
 
   scenario "user sees a maximum of 12 users per page" do
-    15.times { FactoryGirl.create(:user, username: Faker::Name.name) }
+    15.times { FactoryGirl.create(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.